mixer/cmd/root.go

```diff
@@ -205,10 +219,12 @@ func fail(err error) {
 		pprof.StopCPUProfile()
 	}
 	log.Printf("ERROR: %s\n", err)
```

Comment:
Replace these  `log.` statements with our new log package.

Reply:
yes, we will need to replace all log.Printf statements. If this flow looks fine, I can replace all log.Printlf statements with new logging and rename the logging package to log.